db/migrate/20231222201909_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePost < ActiveRecord::Migration[7.1]
```

Comment:
- [ ] **[OPTIONAL]** There is an issue with your migration. There is a mismatch in the file name and the class name. This should be `CreatePosts`

  ![image](https://github.com/Sami-Ullah-Tufail/blog/assets/36542986/de37bb67-ae29-4bfb-b374-e95f6cf5299e)